WordPress/gutenberg
# [Comments Query Loop] Show placeholder comments on site editor

## Description
This is a quick approach to #37143. We are using the default placeholders' fallback of all components of a comment template.
A better solution may be to update all fallbacks to not depend on getting an empty comment object, feel free to comment or discuss if we should work on a better approach.

## How has this been tested?



- Go to site editor.
- Add a Comment Query Loop.
- Check that we are showing the default placeholders.
- Go to a post or a page.
- Add a Comment Query Loop.
- Check that comments loaded are real ones.

## Screenshots 
Screenshots were done on site editor.
Before: 
![commentsBeforePR](https://user-images.githubusercontent.com/37012961/150137100-f0a4009e-4696-4c35-80df-364392c3c3c8.png)

After:

![Screenshot 2022-01-27 at 15 06 00](https://user-images.githubusercontent.com/37012961/151374813-15084512-f96a-4e7e-81ec-2d67aa80b001.png)



## Types of changes





New possible feature discussed on #37143

## Checklist:
